packages/javascript/bh-shared-ui/src/components/CreateSAMLProviderForm/CreateSAMLProviderForm.tsx

```diff
@@ -103,18 +103,21 @@ const CreateSAMLProviderForm: React.FC<{
                                 <Box p={1} borderRadius={4} bgcolor={theme.palette.neutral.tertiary}>
                                     <Box display='flex' flexDirection='row' alignItems='center'>
                                         <Button variant='secondary'>
-                                            Choose File
-                                            <input
-                                                hidden
-                                                type='file'
-                                                accept='.xml'
-                                                value={fileValue}
-                                                onChange={(e) => {
-                                                    setFileValue(e.target.value);
-                                                    field.onChange(e.target.files as FileList);
-                                                }}
-                                                onBlur={field.onBlur}
-                                            />
+                                            <>
```

Comment:
from the look so it the fragments are not needed?

Also, no need to explain why this change, I just want to double check that its intentionally in this change set?

Reply:
Good point! Removing the fragment wrapping. I believe I made this change to patch up the unit test as it was looking for a label element.